server/channels/api4/scheduled_post.go

```diff
@@ -16,6 +18,7 @@ import (
 
 func (api *API) InitScheduledPost() {
 	api.BaseRoutes.Posts.Handle("/schedule", api.APISessionRequired(createSchedulePost)).Methods(http.MethodPost)
+	api.BaseRoutes.Posts.Handle("/schedule/{scheduled_post_id:[A-Za-z0-9]+}", api.APISessionRequired(updateScheduledPost)).Methods(http.MethodPut)
```

Comment:
Would it make sense for this to be changed to a patch endpoint instead of an update one? I've historically found patch endpoints to be a bit easier to use since they allow you to just pass around an ID and one or two other fields, and you don't need to worry about potentially overwriting data that's been changed from elsewhere. I don't know how likely it is to conflict, but I wonder if there's a way for users to trigger two changes at once (like changing the time and priority) or how much we care if a user updates a scheduled post as it has been sent by the server

Reply:
> but I wonder if there's a way for users to trigger two changes at once

When we add a better inline editing, user could change message, props, fileIDs, priority all at once. Scheduled time would stay separate anyways.

I don't see much use of patch vs put in our use case from client so leaving it as it. Will update if need arises.